hack/ruleguard/rules.go

```diff
@@ -0,0 +1,38 @@
+// +build ignore
```

Comment:
would putting this in hack/tools/ruleguard keep the dep out of the root go.mod file?

Reply:
tried that, didn't work